e2e/config/config.go

```diff
@@ -119,7 +119,8 @@ type Contracts struct {
 
 // Solana contains the addresses of predeployed contracts on the Solana chain
 type Solana struct {
-	GatewayProgramID string `yaml:"gateway_program_id"`
+	GatewayProgramID string             `yaml:"gateway_program_id"`
+	SPL              DoubleQuotedString `yaml:"spl"`
```

Comment:
Could you please clarify what SPL stands for in this context (in the comment)?  Is it a ticker symbol or a SPL address, or something else entirely?